tests/integration-tests/bundler-css-spec.js

```diff
@@ -0,0 +1,122 @@
+
+describe("Css Bundling:", function() {
+
+    var exec = require('child_process').exec,
+		fs = require('fs'),
+        testHelper = require('./helpers/integration-test-helper.js'),

```

Comment:
This setup section likely needs some work so its magic-ed for you. 

Same with the beforeEach and afterEach.